Source/ARTWebSocketTransport.m

```diff
@@ -55,7 +55,8 @@ + (void)setWebSocketClass:(const Class)webSocketClass {
     configuredWebsocketClass = webSocketClass;
 }
 
-- (instancetype)initWithRest:(ARTRestInternal *)rest options:(ARTClientOptions *)options resumeKey:(NSString *)resumeKey connectionSerial:(NSNumber *)connectionSerial {
+- (instancetype)initWithRest:(ARTRestInternal *)rest options:(ARTClientOptions *)options
```

Comment:
It's either one line, or line per parameter.

Reply:
f94f2a0c84ad4702342edead4deb7b4d0ddc1c69